File: w4/php/square2.php
<?php
    $size = intval (fgets(STDIN));
    $count = '';
    for ( $row = 1; $row <= $size; $row++) {
        $out = '';
        for ( $col = 1; $col <= $size; $col++) {
            $count += 1;
            $out .= $count . ' ';
        }
        echo $out . PHP_EOL;
    }
?>

File: w1/square2.c
#include <stdio.h>

int main() {
    int size;
    int count = 1;

    scanf("%d", &size);
    for ( int row = 1; row <= size; row++ ) {
        for ( int col = 1; col < size; col++ ) {
            printf("%d ", count);
            count += 1;
        }
        printf("%d\n", count);
        count += 1;
    }
    return 0;
}
